runtime/export/layer.go

```diff
@@ -1,9 +1,14 @@
 package export
 
+import (
+	"github.com/wagoodman/dive/dive/filetree"
+)
+
 type layer struct {
-	Index     int    `json:"index"`
-	ID        string `json:"id"`
-	DigestID  string `json:"digestId"`
-	SizeBytes uint64 `json:"sizeBytes"`
-	Command   string `json:"command"`
+	Index     int                 `json:"index"`
+	ID        string              `json:"id"`
+	DigestID  string              `json:"digestId"`
+	SizeBytes uint64              `json:"sizeBytes"`
+	Command   string              `json:"command"`
+	FileList  []filetree.NodeData `json:"fileList"`
```

Comment:
instead of NodeData which shows the view information as well, should this be FileInfo instead?

Reply:
Yes currently, the `FileList` will contain the `ViewInfo`, which can be seen in the testcase here- https://github.com/Akash-Nayak/dive/blob/f1783c55b3d7bc02d5e9cbb681d25044998375cc/runtime/export/export_test.go#L28.

Should I make the changes so that it only contains the `FileInfo`?

Reply:
> instead of NodeData which shows the view information as well, should this be FileInfo instead?

Hi @wagoodman, I have changed it from `NodeData` to `FileInfo`.